component/src/main/java/org/wso2/carbon/extension/analytics/receiver/rabbitmq/internal/util/RabbitMQBrokerConnectionConfiguration.java

```diff
@@ -0,0 +1,124 @@
+/*
+*  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.carbon.extension.analytics.receiver.rabbitmq.internal.util;
+
+import org.wso2.carbon.event.input.adapter.core.InputEventAdapterConfiguration;
+
+/**
+ * Getting the connection values from the adapter configuration.
+ */
```

Comment:
Best practice to follow is adding @since along in the Java doc comments. And the comment should end in with the period otherwise the build would fail in windows.